apps/engineeringblog/app/layout.tsx

```diff
@@ -0,0 +1,29 @@
+import ThemeRegistry from "@/theme/ThemeRegistry";
```

Comment:
Check how we have defined and imported themes in RoboShield, it would be better to have one way of doing things

Reply:
Hi Michael, thanks for the feedback, It's mostly the same except there's a little bit of abstraction with the app router cache provider. If you check the component structure, it's the same, just moves the theme configuration and setup outside the main entrypoint component.

Reply:
Hi Michael, gonna resolve this, @kelvinkipruto used to `StyledRoot` to match with the implementation in Roboshield.